mctx/_src/policies.py

```diff
@@ -309,9 +307,10 @@ def stochastic_muzero_policy(
   # construct a dummy afterstate embedding
   batch_size = jax.tree_util.tree_leaves(root.embedding)[0].shape[0]
   dummy_action = jnp.zeros([batch_size], dtype=jnp.int32)
-  _, dummy_afterstate_embedding = decision_recurrent_fn(params, rng_key,
+  output, dummy_afterstate_embedding = decision_recurrent_fn(params, rng_key,
```

Comment:
You can call it `dummy_output`.
And update the indentation to match the "(".
Sorry for being a perfectionist.